benchmark/README.md

```diff
@@ -60,9 +60,20 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
     <true for sql tests, false for functional tests>
     ```
 
+    **Note**: Ensure that you build the worker and application with .NET Core 3.0 in order to run hardware acceleration queries.
+
+
 ## Python
 1. Upload [run_python_benchmark.sh](run_python_benchmark.sh) and all [python tpch benchmark](python/) files to the cluster.
-2. Run the benchmark by invoking:
+2. Install pyarrow and pandas on all nodes in the cluster. For example, if you are using Conda, you can use the following commands to install them.
+    ```shell
+    export PATH=$PATH:/usr/bin/anaconda/bin
+    sudo chown -R sshuser /usr/bin/anaconda

```

Comment:
Why do you need to change ownership of the files/folders in /usr/bin/anaconda  ?  

Wouldn't it better to run `sudo` with the conda commnads below ?

Reply:
yea that makes sense. @Niharikadutta, can you do a follow up PR?

Reply:
I feel like I faced some issue while using sudo conda, I'll check again and create a follow up PR. Thanks!